client/index.js

```diff
@@ -66,8 +65,8 @@ function listenForTurn(c) {
   });
 }
 
+// set up divs for counter placement events
 function createTopButtons(game) {
-  // set up divs for individual elements
   for (let c = 0; c < game.numCols; c++) {
     $('#top-buttons').append($(`<button>${c}</button>`).addClass('btn btn-secondary')
       .prop('id', `top-button-${c}`));
```

Comment:
Rather than calling `listenForTurn(c);` after this line, which has to search the document for this element, it might be neater to use the current reference to the element more like this:
```suggestion
      .prop('id', `top-button-${c}`))
      .click(() => onTopButtonClick(c));
```